src/sentry/integrations/pagerduty/utils.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import
+
+import logging
+import six
+
+from sentry.incidents.models import IncidentStatus
+from sentry.integrations.metric_alerts import incident_attachment_info
+from sentry.models import PagerDutyService
+from sentry.shared_integrations.exceptions import ApiError
+
+from .client import PagerDutyClient
+
+logger = logging.getLogger("sentry.integrations.pagerduty")
+
+
+def build_incident_attachment(incident, integration_key, metric_value=None):
+    data = incident_attachment_info(incident, metric_value)
+    if incident.status == IncidentStatus.CRITICAL.value:
+        severity = "critical"
+    elif incident.status == IncidentStatus.WARNING.value:
+        severity = "warning"
+    elif incident.status == IncidentStatus.CLOSED.value:
+        severity = "info"
+
+    footer_text = "Sentry Incident | {}".format(data["ts"].strftime("%b %d"))
+
+    return {
+        "routing_key": integration_key,
+        "event_action": "trigger"
+        if incident.status in [IncidentStatus.WARNING.value, IncidentStatus.CRITICAL.value]
+        else "resolve",
+        "dedup_key": "incident_{}".format(incident.identifier),
```

Comment:
@ceorourke So the `identifier` increments and is unique to an org. This could cause a problem if someone ever tries to use the pagerduty account with multiple Sentry organizations since the IDs will overlap.

Reply:
@scefali what about something like `incident_{incident.identifier}_{org.id}`?

Reply:
I'd put the `org_id` before the identifier but yea, looks good!